### DISCUSSION TITLE
Read a JDFFile in chunks

### DISCUSSION THREAD

**Discussion by gcalderone** [xiaodaigh/JDF.jl#71] [Ideas]
Is it possible (or will someday be possible) to read a JDFFile in chunks?

I.e., something similar to:
```julia
df = DataFrame(JDF.load("iris.jdf", start=2, length=2))
```
to read just the 2nd and 3rd row.

Thanks!

**Comment by xiaodaigh** [comment]
Yeah. Might be possible. I am thinking through it. But for small ranges, it's not gonna be super efficient.

**Reply by gcalderone** [reply]
> But for small ranges, it's not gonna be super efficient.

The main reason to read in chunks is to allow handling of DataFrames which would not fit in memory, so I guess it would not be a problem if the performance is not the best in reading a few lines...

**Reply by xiaodaigh** [reply]
yeah. Agreed.
